packages/amplify-category-api/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.js

```diff
@@ -372,8 +447,165 @@ async function askSecurityQuestions(context, parameters) {
   const advancedSettingsAnswer = await inquirer.prompt([advancedSettingsQuestion]);
 
   if (advancedSettingsAnswer.advancedSettings) {
+    authConfig = await askAdditionalAuthQuestions(context, parameters, authConfig, defaultAuthType);
+    if(process.env.SYNC === "true") {
+      resolverConfig = await askResolverConflictQuestion(context, parameters, modelTypes);
+    }
+  }
+
+  return { authConfig, resolverConfig };
+}
+
+async function askResolverConflictQuestion(context, parameters, modelTypes) {
```

Comment:
Why are we collecting ```modelTypes``` earlier in the process? Are we using them even if the answer is 'N' to the question?